app/renderer/components/ServerMonitor/ServerMonitor.js

```diff
@@ -5,6 +5,8 @@ import { STATUS_RUNNING, STATUS_STOPPING,
 import styles from './ServerMonitor.css';
 import AnsiConverter from 'ansi-to-html';
 
+import AppiumSmallMagenta from '../../images/appium_small_magenta.png';
```

Comment:
Parcel handles importing of images by returning a path to that image. The file located at `app/renderer/images/appium_small_magenta.png` will be copied over to `dist/` and this module will load that file.

Reply:
so parceljs has an image import loader or something that dynamically updates the `AppiumSmallMagenta` name based on the build?

Reply:
It copies `images/appium-small-magenta.png` to `dist/` (along with the other assets )and renames it `appium-small-magenta.<cache-bust>.png`.

When `ServerMonitor.js` imports `AppiumSmallMagenta` it imports the name of that file as a string that can be passed into the the `src` attribute of `<img>`